source/platforms/coiled.md

```diff
@@ -1,65 +1,142 @@
 # Coiled
 
-You can deploy RAPIDS on a multi-node Dask cluster with GPUs using [Coiled](https://www.coiled.io/).
+You can deploy RAPIDS on cloud VMs with GPUs using [Coiled](https://www.coiled.io/). Coiled is a software platform that manages Cloud VMs on your behalf. It manages software environments and can launch Python scripts, Jupyter Notebook servers, Dask clusters or even just individual Python functions. Remote machines are booted just in time and shut down when not in use or idle.
 
 By using the [`coiled`](https://anaconda.org/conda-forge/coiled) Python library, you can setup and manage Dask clusters with GPUs and RAPIDs on cloud computing environments such as GCP or AWS.
 
-Coiled clusters can also run Jupyter which is useful if you don't have a local GPU.
+## Setup
 
-## Quickstart
+Head over to [Coiled](https://docs.coiled.io/user_guide/setup/index) and register for an account.
 
-### Login
+Once your account is set up, install the coiled Python library/CLI tool.
 
-To get started you need to install Coiled and login.
+```bash
+pip install coiled
+```
 
-```console
-$ conda install -c conda-forge coiled
-$ coiled login
+Then you can authenticate with your Coiled account.
+
+```bash
+coiled login
 ```
 
-For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/getting_started.html).
+For more information see the [Coiled Getting Started documentation](https://docs.coiled.io/user_guide/setup/index).
 
-### Software Environment
+## Notebook Quickstart
 
-Next you'll need to register a RAPIDS software environment with Coiled.
+The simplest way to get up and running with RAPIDS on Coiled is to launch a Jupyter notebook server using the RAPIDS notebook container.
 
-You can either build this from the official RAPIDS Container images.
+```bash
+coiled notebook start --gpu --container {{ rapids_notebooks_container }}
+```
 
-```python
-import coiled
+![Screenshot of Jupyterlab running on Coiled executing some cudf GPU code](../_static/images/platforms/coiled/coiled-jupyter.png)
 
-coiled.create_software_environment(
-    name="rapids-{{ rapids_version.replace(".", "-") }}",
-    container="{{ rapids_container }}",
-)
+## Software Environments
+
+By default when running remote operations Coiled will [attempt to create a copy of your local software environment](https://docs.coiled.io/user_guide/software/sync.html) which can be loaded onto the remote VMs. While this is an excellent feature it's likely that you do not have all of the GPU software libraries you wish to use installed locally. In this case we need to tell Coiled which software environment to use.
```

Comment:
Would launching via the notebooks container also imply that you don't need a GPU locally neither?  If so add a note about that can be helpful.